app/components/Onboarding/NewWalletPassword.scss

```diff
@@ -35,3 +35,10 @@
     visibility: visible;
   }
 }
+
+.helpMessage {
```

Comment:
currently we highlight the bottom border of the input that isn;t correct and make the border `$red`. we also make the error text `$red`. can you make those style changes?

Reply:
Since we use input field with all borders making just the bottom one red will look strange. I haven't seen this pattern in sites/apps, so in my opinion it will not be consistent in usability point of view. I've made all borders and error text to get $red when error occurs. Is that Ok?